so_pysm_models/get_so_model.py

```diff
@@ -0,0 +1,102 @@
+import healpy as hp
+import numpy as np
+import pysm
+import os
+from astropy.utils import data
+from pysm.common import read_map, loadtxt
+
+data_dir = os.path.join(os.path.dirname(__file__), 'so_template')
+template = lambda x: os.path.join(data_dir, x)
+DATAURL = "http://portal.nersc.gov/project/cmb/so_pysm_models_data/"
+
+def get_data_from_url(filename):
+    with data.conf.set_temp("dataurl", DATAURL), data.conf.set_temp(
+        "remote_timeout", 30):
+        map_out = data.get_pkg_data_filename(filename)
+    return map_out
+
+def models(key, nside, pixel_indices=None, mpi_comm=None, small_scale=False):
+     model = eval(key)(nside, pixel_indices=pixel_indices, mpi_comm=mpi_comm, small_scale=small_scale)
+     for m in model:
+         m['pixel_indices'] = pixel_indices # include pixel indices in the model dictionary
+         m['nside'] = nside
+     return model
+
+def SO_d0(nside, pixel_indices=None, mpi_comm=None, small_scale=False):
+    if small_scale:
```

Comment:
what about choosing automatically based on `nside`? do you see any issue with that?

Reply:
Yes, we could do that. The reason why I was leaving the `small_scale` option to user is just because there might be a case where you need maps with high Nside but don't really care about having small scales on it, in this case the code is much faster if you pick the Nside=512 template.

Reply:
ok